tests/data_context/test_config.py

```diff
@@ -0,0 +1,62 @@
+from great_expectations.data_context import BaseDataContext
+from great_expectations.data_context.types.base import (
+    ConcurrencyConfig,
+    DataContextConfig,
+    InMemoryStoreBackendDefaults,
+    ProgressBarConfig,
+)
+
```

Comment:
Couldn't find a good location for tests so I took the existing test_concurrency_config and made it more general